### ISSUE TITLE
But how do I actually do that?

### ISSUE THREAD

**Issue by rbwhitaker** [NuGet/docs.microsoft.com-nuget#1502]
I got this error (NU5005), but this documentation is not very helpful. I don't know what to do with it. I can see a .pdb file next to my .dll after compiling. It seems to have correctly created the .nupkg file. I'm new to this symbol package thing, and I just don't know where to even go with this. It seems like a link to how to make sure that your source/symbol files are in the right spot would be helpful.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b0a9f98c-5fde-8c3b-01d4-2aa37e749403
* Version Independent ID: 58213a68-28d4-8e95-534d-33b1adf4ed53
* Content: [NuGet Error NU5005](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5005)
* Content Source: [docs/reference/errors-and-warnings/NU5005.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/reference/errors-and-warnings/NU5005.md)
* Product: **nuget**
* GitHub Login: @mishra14
* Microsoft Alias: **karann**

**Comment by rbwhitaker** [comment]
So I was finally able to get this to work... at least I think I was. In the .nuspec file, I added both the DLL and the PDB file. When I told it to compile and also compile the symbols into a .snupkg, it seems as though only the DLL ended up in the .nupkg, and the PDB file only ended up in the .snupkg file. I can't say that this feels totally natural. My natural inclination is that with both listed in the .nuspec file, that both would end up in at least the .nupkg file. I don't care if my natural inclination is right, but I'm having a very hard time finding any sort of documentation anywhere that specifically calls this out. It seems like if that's actually how it works, it would be helpful if either this page (which is where I ended up at when I did it wrong) or the page about making .snupkg files (https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg) called out this behavior. Anyway... just thought I'd post my results in case it helps anybody else who ends up here with a similar problem.
